R/project.R

```diff
@@ -270,24 +270,24 @@ renv_project_synchronized_check <- function(project = NULL, lockfile = NULL) {
   missing <- setdiff(lockpkgs, basename(libpkgs))
   if (length(missing)) {
     msg <- lines(
-      "* One or more packages recorded in the lockfile are not installed.",
-      "* Use `renv::status()` for more details."
+      "- One or more packages recorded in the lockfile are not installed.",
+      "- Use `renv::status()` for more details."
     )
     writef(msg)
     return(FALSE)
   }
 
   # otherwise, use status to detect if we're synchronized
-  info <- quietly({
+  info <- local({
```

Comment:
This is really the guts of the PR — previously this didn't create a new scope, so the message about the project being out of sync was never shown.